chromadb/api/segment.py

```diff
@@ -159,11 +161,24 @@ def create_collection(
 
         id = uuid4()
 
-        coll, created = self._sysdb.create_collection(
+        model = CollectionModel(
             id=id,
             name=name,
             metadata=metadata,
+            configuration=configuration
+            if configuration is not None
```

Comment:
Ok so this is where the default behavior is handled, in that case I don't understand why ServerAPI doesn't default it?

Reply:
I think you're probably right, let's discuss briefly.